sdk/storage/Azure.Storage.DataMovement/src/StorageResourceReadStreamResult.cs

```diff
@@ -44,7 +44,7 @@ public class StorageResourceReadStreamResult
             StorageResourceItemProperties properties)
         {
             Content = content;
-            ContentLength = range.Length;
+            ContentLength = range != default ? range.Length : 0;
```

Comment:
nit but null coalesce is generally clearer `range?.Length ?? 0`

Reply:
`HttpRange` isn't nullable. When I tried to do `range?.Length` it says "Operator '?' cannot be applied to operand of type 'HttpRange'."